io/zenoh-transport/src/unicast/establishment/accept.rs

```diff
@@ -684,10 +687,13 @@ pub(crate) async fn accept_link(link: LinkUnicast, manager: &TransportManager) -
         };
     }
 
+    #[allow(clippy::unnecessary_min_or_max)]
```

Comment:
```suggestion
    // Clippy raises a warning because `batch_size::UNICAST` is currently equal to `BatchSize::MAX`.
    // However, the current code catches the cases where `batch_size::UNICAST` is different from `BatchSize::MAX`.
    #[allow(clippy::unnecessary_min_or_max)]
```